lib/food_truck.rb

```diff
@@ -0,0 +1,24 @@
+class FoodTruck
+  attr_reader :name, :inventory, :stock
+
+  def initialize(name)
+    @name = name
+    @inventory = {}
+  end
+
+  def check_stock(item)
```

Comment:
This logic will not work in the situations where the inventory hash is not empty, but does not contain the requested item.

The method is also not working because you are looking for the wrong key.  You should be looking for the key `item`